packages/web-react/src/components/UNSTABLE_Header/README.md

```diff
@@ -0,0 +1,177 @@
+# UNSTABLE Header
+
+> ⚠️ This component is UNSTABLE. It may significantly change at any point in the future.
+> Please use it with caution.
+
+The `UNSTABLE_Header` component is planned to replace the `Header` component in the future.
+
+This component provides these components:
+
+- [UNSTABLE_Header](#unstable-header)
+- [UNSTABLE_HeaderLogo](#unstable-headerlogo)
+
+## UNSTABLE Header
+
+The `UNSTABLE_Header` component is a main wrapper which provides mainly the visual for the Header.
+
+```jsx
+import { UNSTABLE_Header } from '@lmc-eu/spirit-web-react';
+
+<UNSTABLE_Header>{/* Content go here */}</UNSTABLE_Header>;
+```
+
+It also sets CSS variable for the Header height which can be used in other nested components.
```

Comment:
```suggestion
It also sets CSS variable for the Header height which can be used by nested components.
```

I had to read it twice, this is clearer, I believe.

Reply:
changed like the suggestion in web